lib/core-server/src/dev-server.ts

```diff
@@ -32,6 +32,10 @@ export async function storybookDevServer(options: Options) {
   app.use((req, res, next) => {
     res.header('Access-Control-Allow-Origin', '*');
     res.header('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept');
+    // These headers are required to enable SharedArrayBuffer
+    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer
+    res.header('Cross-Origin-Opener-Policy', 'same-origin');
```

Comment:
@darthtrevino this caused me havoc on friday.
This exact example or any https:image file here got hit with cors.
```
// .storybook/YourTheme.js

import { create } from '@storybook/theming';

export default create({
  base: 'light',
  brandTitle: 'My custom storybook',
  brandUrl: 'https://example.com',
  brandImage: 'https://place-hold.it/350x150',
});
```
This also got hit with cors 
<script src="https://cdn.tailwindcss.com/"></script>

Error in chrome and firefox -> "Failed to load resource: net::ERR_BLOCKED_BY_RESPONSE.NotSameOriginAfterDefaultedToSameOriginByCoep"

I traced it to my yarn lock and then traced it from alpha 45 -> to alpha 50.

Reply:
Overall not sure why but inside of storybook these headers stop getting attached. (Yes I see the config above.. looking into express)
access-control-allow-origin: *
access-control-expose-headers: *

Reply:
If my understanding is right maybe you can add COOP and COEP as a flag for story book instead, just like ssl is a flag. That way if you need shared array buffer, you will have to deal with the reality that and external resource that does not support CORS will not work.. WHICH is working as expected. However IMO I find this to be more of an edge case and therefore fits better as a flag with some caveat text around it.